cutils.c

```diff
@@ -612,6 +612,7 @@ void rqsort(void *base, size_t nmemb, size_t size, cmp_f cmp, void *opaque)
 }
 
 #if defined(_MSC_VER)
+struct timezone;
```

Comment:
The `include <time.h>` should pull this in, right? Why doesn't that work with clang?